src/css/controls/imports/slider.less

```diff
@@ -120,24 +127,25 @@
     align-items: center;
     background: transparent none;
     padding: 0 12px;
+}
+
+.jw-slider-time .jw-cue {
+    .jw-slider-components();
+    background-color: @background-color;
+    cursor: pointer;
+    position: absolute;
+    width: 6px;
+}
+
+.jw-slider-time,
+.jw-horizontal-volume-container {
     z-index: 1;
     outline: none;
 
     .jw-rail,
     .jw-buffer,
-    .jw-progress,
-    .jw-cue {
-        backface-visibility: hidden;
-        height: 100%;
-        transform: translate(0, -50%) scale(1, 0.6);
-        transition: transform 150ms ease-in-out;
```

Comment:
Leave this part. It more efficient to share these styles (is there a way in less to omit the .jw-horizontal-volume-container > cues line?)

Reply:
I tried to find a way to omit `.jw-horizontal-volume-container > cues`, but couldn't find any solution yet